src/components/TabPanel.js

```diff
@@ -28,17 +28,25 @@ type TabPanelProps = {
    * class assigned, for use in nested search results.
    */
   nested: boolean;
+  /**
+   * Property that contains the prefix for data-test attribute
+   * added to elements to be uniquely identified by testing tools
+   * like Selenium.
```

Comment:
In the comment, you should say what will be appended to the prefix to create the final value of the data-test attribute. For example, the `id` field of the TabInfo object for each tab in this case. This way someone will know that, given the prefix "inspectorTab-" and the tabs "a," "b," and "c," the actual data-test attributes will be "inspectorTab-a," "inspectorTab-b," and "inspectorTab-c."